ansible/roles/products/hand-e/server/desktop-icons/templates/scripts/start-docker-container-only-nuc.j2

```diff
@@ -0,0 +1,6 @@
+#jinja2: trim_blocks:False
+#!/bin/bash
+NUC_NAME={{ nuc_username }}
+NUC_ADDRESS={{ nuc_address }}
+CONTAINER={{ container_name }}
+ssh -X $NUC_NAME@$NUC_ADDRESS "docker exec --user user $CONTAINER bash -c 'source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash;export DISPLAY=localhost:10.0;{{ command_to_run }} 2> >(tee -a /home/user/.ros/log/stderr.log >&2)'"
```

Comment:
why export DISPLAY=localhost:10.0
What if its another number on the nuc? On mine it was 12
I think you dont have to export DISPLAY at all

Reply:
I tried without exporting DISPLAY and it didn't work (if I don't export it, the DISPLAY value will be 0.0 always when I run this as part of a script - I know you got the "right" DISPLAY when you do it manually using a terminal - but doing it with a script causes it to behave differently). This was the only way I could make it work. The number is always localhost:10.0 if no other users are ssh -X'ed to the NUC. That's because      X11DisplayOffset
             Specifies the first display number available for sshd(8)'s X11
             forwarding.  This prevents sshd from interfering with real X11
             servers.  The default is 10.
and the display number starts from 0. The reason you had 12 was because there was 2 other ssh -X sessions to the NUC. In all my testing in different NUCs, in normal scenarios (e.g. you only have 1 ssh -X session open with the NUC at any given time), localhost:10.0 works.